doc/whats_new.rst

```diff
@@ -539,6 +539,12 @@ API changes summary
      - ``utils.stats.rankdata``
      - ``neighbors.approximate.LSHForest``
 
+    - Deprecate the ``y`` parameter in `transform` and `inverse_transform`.
+      The method  should not accept ``y`` parameter, as it's used at the prediction time.
+      :issue:`8174` by :user:`Tahar Zanouda <tzano>`, :user:`Alexandre Gramfort`_
+      and :user:`Raghav RV`_.
```

Comment:
I doubt this renders fine for you and me

I think for me it should only be `Alexandre Gramfort`_ as below